domain/storage/service/storage.go

```diff
@@ -56,15 +56,22 @@ func (s *StorageService) ImportFilesystem(ctx context.Context, arg ImportStorage
 		return "", errors.Capture(err)
 	}
 
-	poolDetails, err := s.st.GetStoragePoolByName(ctx, arg.Pool)
-	if errors.Is(err, storageerrors.PoolNotFoundError) {
+	poolUUID, err := s.st.GetStoragePoolUUID(ctx, arg.Pool)
```

Comment:
We are going to re-write this func in an upcoming JIra task as the function should be trying to work out if this request was for a pool or provider.

Can I suggest just commenting out this func for the time being that way the implementation here isn't affecting the design decisions of this PR ?